test/TestProjects/MgmtKeyvault/tests/Generated/Mock/DeletedManagedHsmCollectionTest.cs

```diff
@@ -22,16 +21,5 @@ public DeletedManagedHsmCollectionMockTests(bool isAsync) : base(isAsync, Record
             ServicePointManager.ServerCertificateValidationCallback += (sender, cert, chain, sslPolicyErrors) => true;
             Environment.SetEnvironmentVariable("RESOURCE_MANAGER_URL", $"https://localhost:8443");
         }
-
-        [RecordedTest]
-        public async Task Get()
```

Comment:
Ignore all the changes in MgmtKeyvault. Mock tests in it has some issues and will be fixed later.

Reply:
testmodeler is updated and MgmtKeyvault Mock tests are fixed.